src/pages/CourseInfoPage/CourseInfoPage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import CourseInfo from 'src/components/CourseInfo/CourseInfo';
+
+const CourseInfoPage = () => {
+	return (
```

Comment:
you can use implicit return here as well

Reply:
It's a matter of preference, but in most projects developers go with implicit returns when there is no additional logic in components